### PR TITLE
Dark mode banner color fix [squashed]

### PR DESCRIPTION
**Pull request by samfun123**
On the side of the banner their is a small purple bar but it doesn't have a consistent width. The explanation button on the banner is also not colored correctly for dark mode.

Without fix:
image

With fix:
image

### CODE REVIEW

File: style/stylesheets/betterttv-dark.css [hunk 1]
```diff
@@ -1969,6 +1969,12 @@ color: white;
 
 .player-column .brick {
     background-color: #222;
+    border-left: 5px solid #6441a5 !important;
+	box-shadow: 0px 0 0 #6441a4 inset !important;
```
Comment (night) [inline style/stylesheets/betterttv-dark.css]: formatting

File: style/stylesheets/betterttv-dark.css [hunk 2]
```diff
@@ -1969,6 +1969,12 @@ color: white;
...
+}
+
+.what {
```
Comment (night) [inline style/stylesheets/betterttv-dark.css]: `.what` is very nonspecific. What if that matches something else on the site and messes it up?

File: style/stylesheets/betterttv-dark.css [hunk 3]
```diff
@@ -1969,6 +1969,12 @@ color: white;
 
 .player-column .brick {
     background-color: #222;
+    border-left: 5px solid #6441a5 !important;
+    box-shadow: 0px 0 0 #6441a4 inset !important;
+}
+
+.what {
```
Comment (samfun123) [inline style/stylesheets/betterttv-dark.css]: .what is used in the default twitch CSS so I followed that.

File: style/stylesheets/betterttv-dark.css (hunk 2 above)
Comment (samfun123) [inline style/stylesheets/betterttv-dark.css]: ``.what`` is what twitch uses in their CSS, so I just followed that.

**Pull request merged**